packages/stripe_platform_interface/lib/src/method_channel_stripe.dart

```diff
@@ -101,6 +102,19 @@ class MethodChannelStripe extends StripePlatform {
         .parse(result: result!, successResultKey: 'paymentMethod');
   }
 
+  @override
+  Future<PaymentMethod> createPaymentMethodWithElements() =>
```

Comment:
I do not prefer to add this method as we really want to stay close to the stripe react native implementation here.